api/document/serializers/content.py

```diff
@@ -82,14 +83,31 @@ def nest_annotations(annotations: Iterator[Annotation],
 class NestedAnnotationSerializer(serializers.Serializer):
     """Figures out which AnnotationSerializer to use when serializing
     content."""
-    serializer_mapping = defaultdict(
-        lambda: BaseAnnotationSerializer,   # will raise an exception when used
-    )
```

Comment:
Note that I removed `defaultdict` here, mainly because having `CONTENT_TYPE` be a dynamic property seemed to confuse mypy, so I just made it a regular ol' string.  `serializer_mapping` only has one call site, in `to_representation`, so I just had the code there check for the existence of the key and raise the same `NotImplementedError` that the old `CONTENT_TYPE` access used to raise.